etcdctl/ctlv3/command/del_command.go

```diff
@@ -46,6 +48,12 @@ func NewDelCommand() *cobra.Command {
 func delCommandFunc(cmd *cobra.Command, args []string) {
 	key, opts := getDelOp(args)
 	ctx, cancel := commandCtx(cmd)
+	getResp, err := mustClientFromCmd(cmd).Get(ctx, key, append(opts, clientv3.WithCountOnly())...)
```

Comment:
I think that proposal was only to make the command that uses '2 arguments' syntax safer, 
it is: 
a) introduce flag to be explicit about removing keys in a range. 
b) steer user's towards migration to the syntax with the --range. 

In 3.5 we would have only warning and a flag to opt-in.  
In 3.6 we would have some moderate sleep. Still I'm not sure we want to slow-down the operation by default by doing 'count' first. We should have flag like '--interactive=false' that disables such overheads.

Reply:
Ok. Fixed.

Reply:
> In 3.5 we would have only warning and a flag to opt-in. 

This PR only add warning. Should I commit PR to main or release-3.5 directly?

> In 3.6 we would have some moderate sleep. Still I'm not sure we want to slow-down the operation by default by doing 'count' first. We should have flag like '--interactive=false' that disables such overheads.

In 3.6. I  will commit another PR.